### ISSUE TITLE
INPC003 should warn when notifying only in one if case.

### ISSUE THREAD

**Issue by JohanLarsson** [DotNetAnalyzers/PropertyChangedAnalyzers#88]
```cs
namespace RoslynSandbox
{
    using System.ComponentModel;
    using System.Runtime.CompilerServices;

    public sealed class Foo : INotifyPropertyChanged
    {
        private int value;

        public event PropertyChangedEventHandler PropertyChanged;

        public int Value => this.value;

        public void Update(bool up)
        {
            if (up)
            {
                this.value++;
            }
            else
            {
                this.value++;
                this.OnPropertyChanged(nameof(this.Value));
            }
        }

        private void OnPropertyChanged([CallerMemberName] string propertyName = null)
        {
            this.PropertyChanged?.Invoke(this, new PropertyChangedEventArgs(propertyName));
        }
    }
}
```
